src/app/ngrx/reducers/episode-downloads.reducer.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { CastleEpisodeDownloadsSuccessAction, CastleEpisodePageSuccessAction } from '../actions';
+import * as fromEpisodeDownloads from './episode-downloads.reducer';
+import { episodes, ep0Downloads, ep1Downloads } from '../../../testing/downloads.fixtures';
+import { EPISODE_PAGE_SIZE } from './models';
+
+describe('EpisodeMetricsReducer', () => {
```

Comment:
Update this description to EpisodeDownloadsReducer to match the rename elsewhere